brush/src/components/InputModal.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import InputROM from './InputROM'
+import { Modal, ModalActions, ModalTitle } from 'former-kit'
+
+const InputModal = () => {
+  const childrenNode = <div>
```

Comment:
When you fix the `children` props, you'll doesn't more need to use `<div>`.
But a tip: in cases that you need to join two or more elements, instead of `<div>`, use [`<Fragments>`](https://reactjs.org/docs/fragments.html).